lib/projects.js

```diff
@@ -18,7 +18,7 @@ const projects = [
       { id: "4", description: "Sand the shelf to smooth out rough edges." },
       { id: "5", description: "Paint or stain the shelf as desired." },
     ],
-  },
+  }, */
```

Comment:
If it's not used, please delete it.

Reply:
oh that was just to test the Empty List Message. We don't have the delete function yet. Thanks for the hint.